blocks/utils/index.js

```diff
@@ -62,3 +62,23 @@ export function isLegacyForm(forms, SelectedFormId) {
 
     return false;
 }
+
+export function isTemplateForm(forms, SelectedFormId) {
+    if (forms) {
+        const data = forms.find((form) => parseInt(form.id) === parseInt(SelectedFormId));
+
+        return data && data.formTemplate !== '';
+    }
+
+    return false;
+}
+
+export function isVisualBuilderForm(forms, SelectedFormId) {
```

Comment:
There is no difference between `isTemplateForm` and `isVisualBuilderForm`. We don't need both functions and looks like `isVisualBuilderForm` is not in use anyway.  Also about the `isLegacyForm` function, in my understanding, all v2 forms have the `formTemplate` property set. So, we can just do something like `return 'legacy' === data.formTemplate `.

Reply:
Resolved: https://github.com/impress-org/givewp/pull/7082/commits/445155c66d0f359fe15b65683a1b0a81b731b8b6